### PR TITLE
First pass at 526 fields

### PR DESCRIPTION
**Pull request by cvalarida**
There are still a _lot_ of questions left to answer, but this is what we (think we) know so far.

### CODE REVIEW

File: dist/21-526EZ-schema.json [hunk 1]
```diff
@@ -2,7 +2,458 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "SUPPLEMENTAL CLAIM FOR COMPENSATION (21-526EZ)",
   "type": "object",
-  "definitions": {},
-  "properties": {},
+  "definitions": {
+    "email": {
```
Comment (jbalboni) [inline dist/21-526EZ-schema.json]: There's a built in email format for json schema, do you know if you need something different than that?

File: src/schemas/21-526EZ/schema.js [hunk 2]
```diff
@@ -1,9 +1,404 @@
...
+        type: 'string'
+      },
+      attachmentType: {
+        type: 'string'
+      },
+      inflightId: {
+        type: 'string'
+      }
+    },
+    militaryPayments: {
```
Comment (jbalboni) [inline src/schemas/21-526EZ/schema.js]: Do you need this nesting? It's generally easier for this schema to be as flat as possible and add nesting on the front end.

File: src/schemas/21-526EZ/schema.js [hunk 3]
```diff
@@ -1,9 +1,404 @@
...
+      type: 'object',
+      properties: {
+        payments: {
```
Comment (jbalboni) [inline src/schemas/21-526EZ/schema.js]: The plural sort of implies there can be more than one, but it's not an array type.

File: src/schemas/21-526EZ/schema.js [hunk 4]
```diff
@@ -1,9 +1,404 @@
...
+      }
+    },
+    directDeposit: {
+      $ref: '#/definitions/directDeposit'
+    },
+    serviceInformation: {
+      type: 'object',
+      properties: {
+        // This is a little different from the common serviceHistory definition
+        servicePeriods: {
```
Comment (jbalboni) [inline src/schemas/21-526EZ/schema.js]: Should this be an array?

File: src/schemas/21-526EZ/schema.js (hunk 2 above)
Comment (cvalarida) [reply src/schemas/21-526EZ/schema.js]: To be honest, I have no idea. This is what it looks like in the excel spreadsheet of the fields. :man_shrugging:

File: dist/21-526EZ-schema.json (hunk 1 above)
Comment (cvalarida) [reply dist/21-526EZ-schema.json]: Oh, I didn't know that. I copied this from `10-10EZ`, I think.

File: src/schemas/21-526EZ/schema.js (hunk 4 above)
Comment (cvalarida) [reply src/schemas/21-526EZ/schema.js]: Yeah, probably...

File: src/schemas/21-526EZ/schema.js (hunk 3 above)
Comment (cvalarida) [reply src/schemas/21-526EZ/schema.js]: No idea. :man_shrugging: 
I'll make it an array and when we run into it, we can change it if needed.

File: dist/21-526EZ-schema.json [hunk 5]
```diff
@@ -2,7 +2,473 @@
...
+            "efctvDt": {
+              "$ref": "#/definitions/date"
+            }
+          }
+        },
+        "homelessness": {
+          "type": "object",
+          "properties": {
+            "hasPointOfContact": {
+              "type": "boolean"
```
Comment (omgitsbillryan) [inline dist/21-526EZ-schema.json]: This is one of the things I wanted to remove but looks to still be in at the moment.  Seems like the concept of "has point of contact" can be deduced by if `pointOfContact` is `null` or not.

File: src/schemas/21-526EZ/schema.js [hunk 6]
```diff
@@ -1,9 +1,425 @@
...
+    // dateRange: definitions.dateRange // hopefully we can use this later
+    phone: definitions.phone,
+    fullPhone: {
+      type: 'object',
+      properties: {
+        areaNbr: {
+          type: 'string'
+          // Validation?
+        },
+        phoneNbr: {
```
Comment (omgitsbillryan) [inline src/schemas/21-526EZ/schema.js]: I requested that abbreviations be removed and EVSS agreed.  `areaNbr` ---> `areaNumber` (or `areaCode`???) & `phoneNbr` ---> `phoneNumber`

Comment (cvalarida) [reply src/schemas/21-526EZ/schema.js]: Excellent! What about `efctvDt`?

Comment (cvalarida) [reply src/schemas/21-526EZ/schema.js]: Judging by the field `pointOfContactAreaCode`, I think it'd make sense to change `areaNumber` to `areaCode`, but I'll wait until we get confirmation.

**Comment by cvalarida** [comment]
Yeah, this is definitely subject to change. I'd like to get it all ironed out before we do _too_ much work on building the form, but we'll use what we have for now so we're not just waiting around.

**Pull request merged**